app/assets/javascripts/components/course_creator/course_creator.jsx

```diff
@@ -413,115 +434,151 @@ const CourseCreator = createReactClass({
       );
     }
 
+    const programs = [
+      {
+        name: 'Generic course',
+        type: 'BasicCourse',
+        description: 'Pastry danish fruitcake liquorice gingerbread. Liquorice pie wafer sesame snaps chupa chups sweet roll. Toffee marzipan sugar plum caramels.'
+      },
+      {
+        name: 'Edit-A-Thon',
+        type: 'Editathon',
+        description: 'Bonbon liquorice tiramisu wafer. Dragée pie caramels gummies muffin. Tart gingerbread biscuit tiramisu. Jelly-o lemon drops icing jujubes liquorice sugar plum jelly beans jelly dragée'
+      },
+      {
+        name: 'Article Scoped Program',
+        type: 'ArticleScopedProgram',
+        description: 'Bonbon liquorice tiramisu wafer. Dragée pie caramels gummies muffin. Tart gingerbread biscuit tiramisu. Jelly-o lemon drops icing jujubes liquorice sugar plum jelly beans jelly dragée'
```

Comment:
This program type only tracks edits for assigned articles or categories. Use this type if you want to limit the statistics to a known collection of articles or topic areas.